tests/unit/matrix/test_assembler.py

```diff
@@ -881,16 +906,16 @@ def _assemble_matrix_(self, request):
         return method_mock(request, Assembler, "_assemble_matrix")
 
     @pytest.fixture
-    def base_margin_(self, request):
-        return instance_mock(request, _UnweightedBaseMargin)
+    def margin_unweighted_base_(self, request):
+        return instance_mock(request, _MarginUnweightedBase)
```

Comment:
This should go in alphabetical order, case-insensitive and ignoring any leading underscore.